From d13a4fd86bf532c59bffe359199e4fbd95dff821 Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Date: Fri, 3 Feb 2012 12:27:49 -0800
Subject: [PATCH 2/3] Revert "ASoC: Don't go through cache when applying
 WM5100 rev A updates"

This reverts commit 78fd75340768980ce6ca646106762a6928289e0c (upstream
commit 495174a8ffbaa0d15153d855cf206cdc46d51cf4) as it breaks the build.

Reported-by: Tim Gardner <rtg.canonical@gmail.com>
Cc: Mark Brown <broonie@opensource.wolfsonmicro.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 sound/soc/codecs/wm5100.c |    2 --
 1 file changed, 2 deletions(-)

diff --git a/sound/soc/codecs/wm5100.c b/sound/soc/codecs/wm5100.c
index c745ce8..42d9039 100644
--- a/sound/soc/codecs/wm5100.c
+++ b/sound/soc/codecs/wm5100.c
@@ -1379,7 +1379,6 @@ static int wm5100_set_bias_level(struct snd_soc_codec *codec,
 
 			switch (wm5100->rev) {
 			case 0:
-				regcache_cache_bypass(wm5100->regmap, true);
 				snd_soc_write(codec, 0x11, 0x3);
 				snd_soc_write(codec, 0x203, 0xc);
 				snd_soc_write(codec, 0x206, 0);
@@ -1395,7 +1394,6 @@ static int wm5100_set_bias_level(struct snd_soc_codec *codec,
 					snd_soc_write(codec,
 						      wm5100_reva_patches[i].reg,
 						      wm5100_reva_patches[i].val);
-				regcache_cache_bypass(wm5100->regmap, false);
 				break;
 			default:
 				break;
-- 
1.7.9.4

